acquia/cli
# CLI-684: Automate CCB ticket creation for pre-releases.

**Proposed changes**

Automatically creates a CCB ticket whenever a pre release is created.

**Alternatives considered**


**Testing steps**


1. Create a pre-release.
2. Check JIRA.

OR, test script locally:
```
export GITHUB_RELEASE_BODY="test body"; export GITHUB_RELEASE_NAME="1.0.0"; export JIRA_HOST="[url]"; export GITHUB_ACTIONS_RUN_URL="[url]"; export PERSONAL_ACCESS_TOKEN="[token]"; export TOKEN_BASED_AUTH="true"; ./.github/workflows/create-ccb-ticket.sh
```

**Merge requirements**
